_posts/2022-03-25-conformation-generation.md

```diff
@@ -0,0 +1,169 @@
+---
+layout: post
+title: Generating Molecular Conformations via Normalizing Flows and Neural ODEs
+authors: Anonymous
+tags: [cheminformatics, drug discovery, generative modeling]  # This should be the relevant areas related to your blog post
+---
+
+1. [Introduction](#introduction)
+2. [An Overview of the Deep Generative Approach](#generative-overview)
+3. [Modeling distributions of distances](#distances)
+4. [Modeling distributions of conformations](#conformations)
+5. [Sampling](#sampling)
+6. [Future Work](#future-work)
+7. [References](#references)
+
+<br>
+<br>
+
+## Introduction
+
+In drug discovery, generating molecular conformations is useful across a variety of applications. For example, docking of various molecular 3D conformations to a specific protein allows drug hunters to decide whether a small molecule binds to a specific pocket in a multitude of conformations or a select few.
+
+<p align="center">
+  <img src="/public/images/2021-12-01-conformation-generation/vina.jpg" style="width: 300px">
```

Comment:
The image paths need to be prefixed with `{{ site.url }}` in order to render correctly when github pages builds the post. You also need to update the path from `2021-12-01-conformation-generation` to `2022-03-25-conformation-generation`(sorry about that change)

For example, this image source should look like:
```
<img src="{{ site.url }}/public/images/2022-03-25-conformation-generation/vina.jpg" style="width: 300px">
```

You'll also need to propagate this change to all other image paths that you've included in your post